homeassistant/components/fritz/device_tracker.py

```diff
@@ -116,6 +116,7 @@ def __init__(self, router: FritzBoxTools, device: FritzDevice) -> None:
         self._router = router
         self._mac = device.mac_address
         self._name = device.hostname or DEFAULT_DEVICE_NAME
+        self._last_activity = device.last_activity
         self._active = False
         self._attrs: dict = {}
```

Comment:
`_attrs` isn't used anymore.

Reply:
Will be removed here: https://github.com/home-assistant/core/pull/50668/commits/dab947294a8352e5f2d40471ca5805ccc3b894f5